aws.yaml

```
@@ -301,9 +301,61 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v9.3.0
+  name: v9.3.1
 spec:
   state: active
+  date: 2020-05-12T11:00:00Z
+  apps:
+    - name: cert-exporter
+      version: 1.2.1
+    - name: chart-operator
+      version: 0.13.0
+    - name: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.1.4
+    - name: coredns
+      componentVersion: 1.6.5
+      version: 1.1.8
+    - name: kube-state-metrics
+      componentVersion: 1.9.5
+      version: 1.0.5
+    - name: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - name: net-exporter
+      version: 1.7.0
```

```suggestion
      version: 1.7.1
```

Same thing for net-exporter but there are some other changes there. So I'm fine if we hold that back.

https://github.com/giantswarm/net-exporter/releases/tag/v1.7.1